Setting/Factions/The Coven.md

```diff
@@ -0,0 +1,32 @@
+---
+tier: "5"
+allies:
+  - "[[Scrappers]]"
+enemies:
+  - "[[Ship Defense Corps]]"
+  - Intership Council
+demonym: cadmus
+aliases:
+  - CADMUS
+---
+#faction
+
+## Background
+They call themselves the Coalition for Biomorphological Equality, but to anyone else they are the Coven, a disorganized group aggressively bent on the democratization of biomodification and freedom to use morphogenic compounds. Those cogent enough often stage violent protests, of [[The Coven|CADMUS]] exclusive research. The [[Ship Defense Corps]] have been known to break up such altercations, by whatever means necessary.
+
+## Turf
+They have small encampments in various parts of the ship but the largest by far are [[The Boroughs]] A mire of fungal growths and makeshift housing, interspersed with the occasional 'clinic'. These surround [[The Academy]]
```

Comment:
In the other file it sounded like the academy was more of an idea than a place, but here it is surrounded?

Reply:
I think this was something I forgot to remove as I was brainstorming, At one point I imagined this a little like a BSL4 lab that was surrounded by protesters, but I think that I prefer the idea vs the place.

Reply:
ooh, agreed but protestors are a fun idea. Maybe that could be a smaller faction, "purists" who actively protest or even sabotage usage of biosynth.